internal/internal_worker_base.go

```diff
@@ -344,52 +414,79 @@ func (bw *baseWorker) isStop() bool {
 	}
 }
 
-func (bw *baseWorker) runPoller() {
+func (bw *baseWorker) runPoller(taskWorker scalableTaskPoller) {
 	defer bw.stopWG.Done()
+	// TODO(quinn): with poller autoscaling, this metric doesn't make a lot of sense
+	// since the number of pollers can go up and down.
 	bw.metricsHandler.Counter(metrics.PollerStartCounter).Inc(1)
```

Comment:
Does Go not have `num_pollers`? We should add if not.

Reply:
We do